pom.xml

```diff
@@ -19,7 +19,7 @@
 	<properties>
 		<java.version>11</java.version>
 		<information.model.version>4.0.0</information.model.version>
-		<log4j.version>2.12.1</log4j.version>
+		<log4j.version>2.14.0</log4j.version>
```

Comment:
Any specific reason why this version is bumped?
In ECC, transitive dependency version is 2.12.1
When we include this jar, we will have 2 jars in uber-jar, once 2.12.1 and other 2.14.0

Reply:
Made some changes. Now we only have this dependency here. The version remains 2.14.0 since at the time of writing it's the newest.